locust/templates/index.html

```diff
@@ -103,14 +103,14 @@ <h2>Start new load test</h2>
                     </div>
                     {% if extra_options %}<label>Custom parameters:</label>{% endif %}
                     {% for key, value in extra_options.items() %}
-                        {% if not ((value is none) or (value is boolean)) %}
+                        {% if not ((value[0] is none) or (value[0] is boolean)) %}
                             <label for="{{key}}">{{key}}</label>
-                            <input type="text" name="{{key}}" id="{{key}}" class="val" value="{{value}}" /><br>
+                            <input type="{{'password' if value[1] else 'text'}}" name="{{key}}" id="{{key}}" class="val" value="{{value[0]}}" /><br>
```

Comment:
This is a little too weird/un-obvious. Maybe use a dict instead of a tuple?

Reply:
I agree @cyberw , it even felt wrong as I was typing it. How about a NamedTuple instead? I also threw in help_text there to display in same style as the other default inputs displayed in the UI if provided. Please refer to new commit 64e930399124f19810550561b3cd791b5e6567cd 

Thank you for your consideration